code/game/gamemodes/objective.dm

```diff
@@ -392,13 +465,10 @@ GLOBAL_LIST_INIT(potential_theft_objectives, (subtypesof(/datum/theft_objective)
 			continue
 		if(O.flags & 2)
```

Comment:
Afraid this isn't possible because `THEFT_FLAG_UNIQUE` is defined in `steal_items.dm` which is alphabetically after `objective.dm`.

Reply:
[code\__DEFINES\antagonists.dm](https://github.com/ParadiseSS13/Paradise/blob/master/code/__DEFINES/antagonists.dm) would work for that I think, especially since it's completely empty at the moment.

Reply:
How can that one be empty of all define files... Thanks I'll use that one